@uportal/content-carousel/src/components/ContentCarousel.scss

```diff
@@ -164,8 +165,6 @@
       border-bottom: none;
       border-bottom: var(--cc-description-border-bottom, none);
       display: block;
```

Comment:
This line is the default fallback for the `display: var(--cc-hero-text-display, block);`, it should move as well.

Reply:
Did you mean it's the default fallback for `display: var(--cc-description-display, block);`, and thus should be removed?  `cc-hero-text-display` and `cc-description-display` target different elements...

Reply:
sorry if I wasn't clear

```
display: var(--cc-hero-text-display, block);
```

needs an IE fallback.

```
display: block;
display: var(--cc-hero-text-display, block);
```

Which means display in IE may not be perfect, but will not have a cascading failure.

Reply:
Ah - thx.  I've updated the PR.